test/functionalTest/cases/3008_get_registration/get_reg_using_v2_created_using_v1.test

```diff
@@ -0,0 +1,148 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET registration using APIv2, created using V1
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# At the moment of implementing the functionality of issue #3008, there was no way to create
+# registrations using APIv2 (Nov/Dec 2017), so, to test this new functionality of querying registrations
+# in APIv2, the registrations had to be created using APIv1.
+# Now, one might say, so, as soon as APIv2 implements creation of registrations, this test should be changed to
+# use APIv2 registration creation instead.
+# Sure, but, creating and getting registrations in pure APIv2 will be sufficiently tested in the future.
+# I would prefer to keep this tests as is, creation in APIv1 and getting in APIv2, to test the 'API version cross'.
```

Comment:
Yes, a test like this could be kept after POST /v2/registrations in order to check cross-version works.

However, I'd suggest to add a .test to check invalid id works fine, i.e. trying to GET /v2/registrations/<id> without previous creating anything and check the right error is returned.

Reply:
Fixed in f114c3e